src/python/pants/option/options_integration_test.py

```diff
@@ -27,12 +27,9 @@ def test_invalid_options() -> None:
         "ERROR] Invalid option 'bad_option' under [pytest]",
     ]
 
-    # We error on invalid CLI options before validating the config file.
```

Comment:
We now validate the config file and options together?

Reply:
There was a version where the order was different but it's actually not now, so I have restored this test to how it was.